makefile

```diff
@@ -61,10 +90,26 @@ scripts/nz-buildings-load: scripts/nz-buildings-load.in
 install: $(SQLSCRIPTS) $(SCRIPTS_built)
 	mkdir -p ${datadir}/sql
```

Comment:
Shorter equivalent command using `-R` (recursive):
```
install: $(SQLSCRIPTS) $(SCRIPTS_built)
	mkdir -p ${datadir}/sql
	cp -R sql/* ${datadir}/sql
	mkdir -p ${datadir}/tests/testdata
	cp tests/testdata/*.sql ${datadir}/tests/testdata
	mkdir -p ${bindir}
	cp $(SCRIPTS_built) ${bindir}
```

Yep, should have used that originally.